test_utils/__init__.py

```diff
@@ -135,6 +140,9 @@ def update_course_with_enterprise_context(course, add_utm_info=True, enterprise_
     )
 
     course_runs = course.get('course_runs', [])
+
+    if add_active_info:
+        course['active'] = utils.has_course_run_available_for_enrollment(course_runs)
```

Comment:
@ziafazal yes we are adding `active` value here to **assert** that if we are getting it from API or not.

Reply:
[this](https://github.com/edx/edx-enterprise/blob/master/tests/test_enterprise/api/test_views.py#L837) unit test will assert that.